lists2safebrowsing.py

```diff
@@ -544,31 +585,23 @@ def main():
 
     publish_to_cloud(config, chunknum)
 
-    shavar_prod_lists_branch = {
-      'name': '70',
-      'commit': {
-        'sha': '8ffa3c6d702002e9a357dd0042848c9418e0ef8a',
-        'url': 'https://api.github.com/repos/mozilla-services/shavar-prod-lists/commits/8ffa3c6d702002e9a357dd0042848c9418e0ef8a'
-      },
-      'protected': False
-    }
-    shavar_prod_lists_non_versioning_branch = {
-      'name': 'non-versioning-branch',
-      'commit': {
-        'sha': '8ffa3c6d702002e9a357dd0042848c9418e0ef8a',
-        'url': 'https://api.github.com/repos/mozilla-services/shavar-prod-lists/commits/8ffa3c6d702002e9a357dd0042848c9418e0ef8a'
-      },
-      'protected': False
-    }
-    # get shavar_prod_lists branches from git
-    shavar_prod_lists_branches = [shavar_prod_lists_branch, shavar_prod_lists_non_versioning_branch]
-    for branch in shavar_prod_lists_branches:
-        branch_name = branch.get('name')
-        try:
-            int(branch_name)
-            get_versioned_lists(config, chunknum, version=branch_name)
-        except ValueError as exc:
-            print(branch_name + ' is not a versioning branch')
+    # create and publish versioned lists
+    resp = requests.get(GITHUB_API_URL + SHAVAR_PROD_LISTS_BRANCHES_PATH)
+    if resp:
+        shavar_prod_lists_branches = resp.json()
+        for branch in shavar_prod_lists_branches:
+            branch_name = branch.get('name')
+            ver = version.parse(branch_name)
+            if isinstance(ver, version.Version):
+                get_versioned_lists(config, chunknum, version=branch_name)
+                print('*** Publish Versioned Lists ***')
+                publish_to_cloud(config, chunknum, check_versioning=True)
+                print('*** Revert Configs ***')
+                revert_config(config, branch_name)
+            else:
+                print(branch_name + ' is not a versioning branch')
```

Comment:
nit: Can this `for` loop go into a function? `def publish_versioned_lists(shavar_prod_lists_branches, config, chunknum)` ?